src/css.js

```diff
@@ -273,7 +274,7 @@ jQuery.extend({
 
 			// Convert "+=" or "-=" to relative numbers (#7345)
 			if ( type === "string" && (ret = rrelNum.exec( value )) ) {
```

Comment:
`rfxnum` is different of the old other regex because these both  case: "10px" and "+=10px" match. So I had this verification here: https://github.com/jquery/jquery/pull/2011/files#diff-414c8f59bff0d1b63680b64763d8c529R276